ci/jenkins/test.sh

```diff
@@ -629,6 +629,11 @@ function deliver_antrea {
     echo "====== Delivering Antrea to all Nodes ======"
     docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
     docker save -o flow-aggregator.tar antrea/flow-aggregator:latest
+    container_runtime=$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 ~ role {print $NF}')
+    if [[ "$container_runtime" =~ "containerd" ]]; then
```

Comment:
Why do you want to check container runtime here?

Reply:
What if the container runtime is not contained? The command will fail immediately

Reply:
We added jumper parameter for containerd runtime, for docker runtime there is no such parameter.